### PR TITLE
Newdatetime

### PR DESCRIPTION
**Pull request by ryankapur**
Current code still needs to be reworked tomorrow to figure out the time zone issue. 

I think the negative altitudes are happening because the time is off by 7 or 8 hours, plus I commented out the catching of negative altitudes (ValueError) temporarily as I was merging/modularizing functions --> Location class.

### CODE REVIEW

**Comment by ryankapur** [comment]
Pls take a look at demo.py when you get a chance.

File: python/tracking.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+
+# curr_time = datetime.datetime(2015, 6, 15, 23, 00, 00, tzinfo = datetime.timezone.utc)
+# original_time = curr_time
+print("Local time: ", d)
+
+#Example: JF1
+print()
+jf1 = t.Location("JF1", 45.541718, -122.960381, d, distAO1, distAO2)
```
Comment (tripzero) [inline python/tracking.py:31]: should we use "lat" and "lon" here from the config?

File: python/tracking.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+t.printLocationInfo(jf1)
+jf1.calcTiltingHeight(distAO1)
+jf1.calcPanningHeight(distAO2)
+print()
+
+#From config.json
+print("Current location via config.json:")
+myLoc = t.Location(name, lat, lon, d, distAO1, distAO2)
```
Comment (tripzero) [inline python/tracking.py:39]: ahh, it looks like you do one for JF and one for the config.  I understand now

Comment (ryankapur) [inline python/tracking.py:39]: yea I can delete JF but it's always nice to compare two relative locations for sanity checks.

**Pull request merged**